<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaySetting</class>
 <widget class="QWidget" name="PlaySetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,12">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#listWidget_toolbar{
background:rgb(56,59,78);
 border: none;
}

#widget_connect,
#widget_version,
#widget{
background:rgb(56,59,78);
 border: none;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,8,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_version" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>收银系统</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_toolbar"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_connect" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ScanCodeSetting" name="widget_scanCodeSetting" native="true"/>
      </item>
      <item>
       <widget class="ReminderSetting" name="widget_reminderSetting" native="true"/>
      </item>
      <item>
       <widget class="CashRegisterSetting" name="widget_cashRegisterSetting" native="true"/>
      </item>
      <item>
       <widget class="CommonSetting" name="widget_commonSetting" native="true"/>
      </item>
      <item>
       <widget class="MiscellaneousFunction" name="widget_miscellaneousFunction" native="true"/>
      </item>
      <item>
       <widget class="AdvancedFunction" name="widget_advancedFunction" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CashRegisterSetting</class>
   <extends>QWidget</extends>
   <header>cashregistersetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CommonSetting</class>
   <extends>QWidget</extends>
   <header>commonsetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdvancedFunction</class>
   <extends>QWidget</extends>
   <header>advancedfunction.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MiscellaneousFunction</class>
   <extends>QWidget</extends>
   <header>miscellaneousfunction.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReminderSetting</class>
   <extends>QWidget</extends>
   <header>remindersetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScanCodeSetting</class>
   <extends>QWidget</extends>
   <header>scancodesetting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
